cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/vsphere/vsphere-mc-common/VsphereClusterSettingsStep.tsx

```diff
@@ -108,13 +114,15 @@ export function VsphereClusterSettingsStep(props: Partial<StepProps>) {
     };
 
     const onFieldChange = (data: string, field: VSPHERE_CLUSTER_SETTING_STEP_FIELDS) => {
-        if (handleValueChange) {
-            handleValueChange(INPUT_CHANGE, field, data, currentStep, errors);
-            setValue(field, data, { shouldValidate: true });
-        }
+        vsphereDispatch({
+            type: INPUT_CHANGE,
+            field,
+            payload: data,
+        } as FormAction);
     };
 
     const onClusterNameChange = (clusterName: string) => {
+        console.log(errors);
```

Comment:
nit: Thinking this logging was for debug purposes and should be taken out?

Reply:
good catch. removed the line.